### DISCUSSION TITLE
umi可以接入Capacitor/cordova 打包andoird apk吗

### DISCUSSION THREAD

**Discussion by HeiShuXianSheng** [umijs/umi#7702] [Q&A]
umi项目平时是用于web端，但是特殊需求需要打一个手机app，有办法快速打包apk吗？

**Comment by tankpanv** [comment]
问一下你解决这个问题了吗

**Reply by HeiShuXianSheng** [reply]
是自己手动在项目中再创建一个cordova项目，然后让cordova来打包build后的静态文件。

**Comment by xiaohuoni** [comment]
可以用这个插件 https://www.npmjs.com/package/@alitajs/cordova
